DTS-STN/canadian-dental-care-plan
# Moving utils server modules to `.server` directory

### Description
Not an incremental PR.

Similar to #2604.

### Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/2406820d-c5db-4853-a6dd-8f956de67a67)

### Checklist

### Test Instructions
Run the application and verify that nothing breaks for apply, renew, and status flows.
